test/firtool/split-verilog.mlir

```diff
@@ -1,13 +1,10 @@
 // RUN: firtool %s --format=mlir -verilog | FileCheck %s --check-prefix=VERILOG
-// RUN: firtool %s --format=mlir -split-verilog -o=%t | FileCheck %s --check-prefix=FIRTOOL
+// RUN: firtool %s --format=mlir -split-verilog -o=%t
```

Comment:
Does this need to `rm -f` the files before this runs to ensure that new files are _actually_ generated and not ones from old runs?

Reply:
Yeah that's actually an excellent point!